### PR TITLE
Convert identifiers package to TypeScript

### PR DESCRIPTION
**Pull request by nawatts**

### CODE REVIEW

File: packages/identifiers/package.json [hunk 1]
```diff
@@ -15,8 +15,12 @@
     "access": "public"
   },
   "scripts": {
-    "build": "gnomad-browser-toolkit-scripts build",
+    "build": "rm -rf lib && tsc",
```
Comment (huy-nguyen) [inline packages/identifiers/package.json:18]: For small packages like this, it's not a problem to use `tsc` to compile TypeScript code but for larger codebase, something like Rollup is better.

File: packages/identifiers/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+    /* Module Resolution Options */
+    "moduleResolution": "node",               /* Specify module resolution strategy: 'node' (Node.js) or 'classic' (TypeScript pre-1.6). */
+    "esModuleInterop": true,                  /* Enables emit interoperability between CommonJS and ES Modules via creation of namespace objects for all imports. Implies 'allowSyntheticDefaultImports'. */
+
+    /* Advanced Options */
+    "skipLibCheck": true,                     /* Skip type checking of declaration files. */
+    "forceConsistentCasingInFileNames": true  /* Disallow inconsistently-cased references to the same file. */
+  },
+  "include": [
```
Comment (huy-nguyen) [inline packages/identifiers/tsconfig.json:33]: Is this necessary?

**Review by huy-nguyen — CHANGES_REQUESTED** [review]

File: packages/identifiers/tsconfig.json (hunk 2 above)
Comment (nawatts) [reply packages/identifiers/tsconfig.json:33]: It turns out that it is not. Will remove.

File: packages/identifiers/src/identifiers.ts [hunk 3]
```diff
@@ -22,12 +22,12 @@ export const parseRegionId = regionId => {
   return { chrom, start, stop }
 }
 
-export const normalizeRegionId = regionId => {
+export const normalizeRegionId = (regionId: string): string => {
```
[the remark this answers is not in the document]
Comment (nawatts) [reply packages/identifiers/src/identifiers.ts:25]: Ooh, did not know that.

File: packages/identifiers/package.json (hunk 1 above)
Comment (nawatts) [reply packages/identifiers/package.json:18]: I've been meaning to ask you about tooling. Let's discuss on Slack.

**Review by huy-nguyen — APPROVED** [review]

**Pull request merged**